src/App.tsx

```diff
@@ -1,36 +1,40 @@
-import React from 'react';
-import { Route, Routes } from 'react-router-dom';
-import './App.css';
-import { Home } from 'components/Home/Home';
-import { CategoryPage } from 'components/Category/CategoryPage';
-import { ProductCard } from 'components/Product/ProductCard';
-import { PaymentFinalization } from 'components/PaymentFinalization/PaymentFinalization';
-import { AdminLogin } from 'components/AdminPanel/AdminLogin';
+import React from 'react'
+import { Route, Routes } from 'react-router-dom'
+import './App.css'
+import { Home } from 'components/Home/Home'
+import { CategoryPage } from 'components/Category/CategoryPage'
+import { ProductCard } from 'components/Product/ProductCard'
+import { PaymentFinalization } from 'components/PaymentFinalization/PaymentFinalization'
+import { AdminLogin } from 'components/AdminPanel/AdminLogin'
+import { OrderSuccess } from 'components/PaymentFinalization/OrderSuccess'
+import { OrderCancel } from 'components/PaymentFinalization/OrderCancel'
 
-import { Orders } from 'components/AdminPanel/Orders';
-import { OnlineChats } from 'components/AdminPanel/OnlineChats';
-import { ResolvedChats } from 'components/AdminPanel/ResolvedChats';
-import { NotFoundPage } from 'components/NotFundPage/NotFoundPage';
-import AdminPrivateRoutes from 'components/AdminPanel/AdminPrivateRoutes';
+import { Orders } from 'components/AdminPanel/Orders'
+import { OnlineChats } from 'components/AdminPanel/OnlineChats'
+import { ResolvedChats } from 'components/AdminPanel/ResolvedChats'
+import { NotFoundPage } from 'components/NotFoundPage/NotFoundPage'
+import AdminPrivateRoutes from 'components/AdminPanel/AdminPrivateRoutes'
```

Comment:
Spróbuj zrobić grupowe exporty/importy. Chodzi o to, żebyś nie miał jak teraz 16 linijek gdzie każdą rzecz importujesz bezpośrednio tylko np w `components` zrób index.ts, który będzie wyglądał mniej więcej tak:
`export * from './Button
export * from ./Category` ( w category jeśli tam jest kilka componentów tak samo index.ts). To pozwoli Ci na importowanie wszystkich komponentów z samego folderu 'components'. Jest to dużo schludniejsze i wygodniejsze

Reply:
done